### PR TITLE
[MIG] attachment_unindex_content: Migration to 16.0

### PR DESCRIPTION
**Pull request by Rodrigosu-Vauxoo**
- Manifest's "Version" updated to "16.0.1.0.0"
- Translation files updated to new version

### CODE REVIEW

**Comment by Rodrigosu-Vauxoo** [comment]
Hello @luisg123v, could you please review this MR please?

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -101,7 +101,7 @@ repos:
       - id: pyupgrade
         args: ["--keep-percent-format"]
   - repo: https://github.com/PyCQA/isort
-    rev: 5.10.1
+    rev: 5.12.0
```
Comment (luisg123v) [inline .pre-commit-config.yaml]: Please avoid this change

File: attachment_unindex_content/i18n/es_AR.po [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* attachment_unindex_content
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 16.0+e\n"
```
Comment (luisg123v) [inline attachment_unindex_content/i18n/es_AR.po]: Update only .pot file, the rest will be automatically regenerated by the bots and Weblate

**Review by luisg123v — CHANGES_REQUESTED** [review]

**Comment by Rodrigosu-Vauxoo** [comment]
Hello @luisg123v , could you review the changes please?

File: attachment_unindex_content/i18n/es_AR.po (hunk 2 above)
Comment (luisg123v) [reply attachment_unindex_content/i18n/es_AR.po]: Not fully applied yet, there still are changes to other translation files other than the `.pot`

**Comment by Rodrigosu-Vauxoo** [comment]
Hello @luisg123v, could you please review the changes?

**Review by luisg123v — APPROVED** [review]
LGTM :+1: 

Hi @moylop260,

Could you review, please?

**Review by moylop260 — APPROVED** [review]
👍

**Comment by luisg123v** [comment]
@moylop260 could you run the command by me, please?

I'm guessing since there are other files modified, the bot doesn't allow me to mark it as migrated.

**Pull request merged**